packages/playwright-core/src/server/registry/dependencies.ts

```diff
@@ -267,6 +267,8 @@ function isSharedLib(basename: string) {
 }
 
 async function executablesOrSharedLibraries(directoryPath: string): Promise<string[]> {
+  if (!fs.existsSync(directoryPath))
```

Comment:
how can this be?

Reply:
sys/lib may or may not be present depending on the platform/wk flavor (gtk/wpe)